packages/driver/src/cypress/cy.ts

```diff
@@ -563,12 +563,9 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
         // we failed setting the remote window props which
         // means the page navigated to a different domain
 
-        // we expect a cross-origin error and are setting things up
-        // elsewhere to handle running cross-domain, so don't fail
-        // because of it
-        if (this.state('readyForMultiDomain')) {
-          signalStable()
-
+        // With multi-domain this is an expected error that may or may not be bad, we will rely on the page load timeout to throw if we don't end up where we expect to be.
+        if (this.config('experimentalMultiDomain') && err.name === 'SecurityError') {
```

Comment:
Is this check going to be different when experimental is turned off or is a SecurityError always going to be accepted in the future?

Reply:
Security Error is the cross origin error, once multi-domain is enabled it's always going to be a possibility.